includes/class-order-manager.php

```diff
@@ -299,6 +299,9 @@ public function create_line_items( $order, $products ) {
         }
 
         $order->save();
+        
+        //New action hook to add custom line items or trigger function after line items are added
+        do_action('after_create_line_items', $order);
```

Comment:
Please add `dokan` prefix before hook name and add proper spacing
```suggestion
        do_action( 'dokan_after_create_line_items', $order );
```